modules/nw-networkpolicy-view.adoc

```diff
@@ -1,25 +1,69 @@
 // Module included in the following assemblies:
 //
 // * networking/network_policy/viewing-network-policy.adoc
-// * networking/configuring-networkpolicy.adoc
 // * post_installation_configuration/network-configuration.adoc
 
 [id="nw-networkpolicy-view_{context}"]
 = Viewing network policies
 
-You can list the network policies in your cluster.
+You can examine the network policies in a namespace.
+
+[NOTE]
+====
+If you log in with a user with the `cluster-admin` role then you can view any network policy in the cluster.
+====
 
 .Prerequisites
```

Comment:
@astoycos, it's possible. Because we're trying to provide a user-story approach to documentation, originally I created each of these as distinct procedures. In retrospect, perhaps that's silly. I don't think it actually _hurts_ anything, though, it's just really verbose.